src/xdocs-test/resources/com/puppycrawl/tools/checkstyle/checks/naming/abstractclassname/XDocExample2.java

```diff
@@ -0,0 +1,15 @@
+/*
+AbstractClassName
+format = (default)^Abstract.+$
+ignoreModifier = true
+ignoreName = (default)false
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.naming.abstractclassname;
+
+abstract class AbstractFifthClass {} // OK
+abstract class SixthClass {} // violation
```

Comment:
All Java example snippets should be identical with only exception for trailing comments.

Reply:
Not possible. We cannot have two classes with the same name in the same package.

Reply:
Unless we declare them as inner class or something. But examples become bigger.

Reply:
Done. I moved them to an inner class.